crates/datastore/src/locking_tx_datastore/datastore.rs

```diff
@@ -1724,8 +1725,14 @@ mod tests {
             );
 
             assert_eq!(
-                schema.indexes,
+                schema
```

Comment:
I think we should also assert that these match the values from `INDEX_IDS` and friends.